build/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Covid Sentinel</title>
-  <script defer src="bundle.js"></script></head>
+  <script defer src="bundle.js"></script><link href="styles.css" rel="stylesheet"></head>
```

Comment:
Nice, just a thing we forgot. Can you delete the build folder and add it to the .gitignore file and then update the pr? We should not commit the webpack output since is something you need to run every time.  Also if that change on line 8 was yours and not from webpack itself you need to import the CSS in the app.jsx file. Webpack should be the responsible for importing it on the html. Thank you and sorry for not realizing it earlier.

Reply:
I'll add the build folder to gitignore and update the pr.  And Yes, webpack added that to index.html. My index.html in the root folder being used as the template does not have those lines, but when you run webpack it outputs a new index.html into build with the script tag for bundle.js and the link for the styles.css.